kumascript/src/info.js

```diff
@@ -141,8 +162,30 @@ const info = {
       tags: tags || [],
       translations: [], //TODO Object.freeze(buildTranslationObjects(data)),
       get subpages() {
+        const parentURL = document.url.toLowerCase();
         return Document.findChildren(document.url)
-          .map((document) => info.getPage(document.url))
+          .map((document) => {
```

Comment:
❤️ This is a really nice improvement over Kuma (which doesn't take into account redirects), because not only do we take into account redirects when returning the immediate children, but we do so intelligently in the sense that we never return redirects that aren't immediate childeren.